frontend/public/components/factory/list-page.jsx

```diff
@@ -331,7 +331,8 @@ export const ListPage = withFallback((props) => {
     textFilter,
     match,
     badge,
-    hideToolbar,
+    hideNameLabelFilters,
```

Comment:
Nit:  put `hideNameLabelFilters` and `hideLabelFilter` on adjoining lines here and elsewhere so you can see they're related?